release.config.js

```diff
@@ -1,18 +1,32 @@
+/* eslint-disable object-property-newline */
 const isCI = require('is-ci');
 // eslint-disable-next-line global-require, no-unused-expressions
 !isCI && require('dotenv').config({path: './.env'});
 
 module.exports = {
   branches: [
-    // eslint-disable-next-line object-property-newline
     {name: 'main', prerelease: 'main'},
     {name: 'master'},
-    // eslint-disable-next-line object-property-newline
     {name: 'canary', prerelease: 'canary'}
   ],
   extends: [],
   plugins: [
-    '@semantic-release/changelog',
+    [
+      '@semantic-release/commit-analyzer',
+      {
+        parserOpts: {
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING']
+        }
+      }
+    ],
+    [
+      '@semantic-release/release-notes-generator',
+      {
+        parserOpts: {
+          noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING']
+        }
+      }
+    ],
```

Comment:
Whoops. Yea, no extending necessary here. Should just be:

```js
    '@semantic-release/release-notes-generator',
```

And to allow access to the repo needs: `@semantic-release/github` (which comes with `semantic-release` and does not need explicitly called out in package.json, however, I personally prefer that).

Fixed in: [`5.0.0-canary.4`](https://github.com/JeromeFitz/git-cz/releases/tag/v5.0.0-canary.4)


![image](https://user-images.githubusercontent.com/3099369/109427489-a5f65c80-79c0-11eb-9b77-518284b65713.png)